🌐 Added Bahasa Indonesia translation for search & portal

rkafp opened the pull request:
Translated/updated all relevant strings in the search & portal  feature to Bahasa Indonesia.

rkafp committed:
🌐 Added Bahasa Indonesia translation for search & portal

cathysarisky reviewed (changes requested):
A quick AI check shows that everything looks reasonable /except/ the name question.  Fix that and we're ready to merge these additions!

rkafp committed:
Updated Name

Localized name selection: opted for Sutan T. Alisjahbana, a Indonesian literary figure.

rkafp committed:
Updated name

cathysarisky commented on ghost/i18n/locales/id/portal.json:
```diff
@@ -77,15 +77,15 @@
-    "jamie@example.com": "",
+    "Invalid email address": "Alamat email tidak valid",
+    "Jamie Larson": "Sutan T. Alisjahbana",
+    "jamie@example.com": "prakasa@example.com",
```
```suggestion
    "jamie@example.com": "sutan@example.com",
```

cathysarisky reviewed (commented):
OK, I suppose that works.  Let's change the example email address to match, please.

rkafp committed:
Updated email address

Co-authored-by: Cathy Sarisky <42299862+cathysarisky@users.noreply.github.com>
